### PR TITLE
Resolve Issue - Find Percentile

### LINKED ISSUE #158: Find Percentile
### **FIND PERCENTILE OF DATA**

Given an 2-D array **_arr_** of size **_m X n_** and an integer _**p**_ lying between 0 and 100 (both inclusive), find:

- The **pth percentile** value of the **flattened array**.
- The **pth percentile** values of **each row**.
- The **pth percentile** values of **each column**.

The **pth percentile** of a dataset gives a number which is **greater than the values of p% elements** of that dataset.

**Example:**
**Input:**
m=3, n=5, p=50
arr=[[3, 8, 1, 4, 6], [1, 7, 4, 6, 12], [[14, 1, 16, 9, 6]]
**Output:**
50th percentile of flattened array is 6.0.
50th percentile values of rows are [4. 6. 9.]
50th percentile values of columns are [3. 7. 4. 6. 6.]

**Hint:** Numpy

### PR DESCRIPTION
**Pull request by himanshusab12**
Completed issue #158

### CODE REVIEW

File: ml/Find_Percentile.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import numpy
+m = int(input("Enter m"))
+n = int(input("Enter n"))
+p = int(input("Enter p"))
+l = []
```
Comment (uttammittal02) [inline ml/Find_Percentile.py]: Try to do without using python list.
Only use `numpy array`

**Review by uttammittal02 — CHANGES_REQUESTED** [review]
Gw

File: ml/Find_Percentile.py (hunk 1 above)
Comment (himanshusab12) [reply ml/Find_Percentile.py]: I have updated the code please check

File: ml/Find_Percentile.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+m = int(input("Enter m"))
+n = int(input("Enter n"))
+p = int(input("Enter p"))
+arr = numpy.zeros((m,n),dtype=int)
+for a in range(m):
+    for b in range(n):
+        arr[a][b]= int(input(f"Enter num"))
+print(arr)
+print(p,"th percentile of flattened array is ", numpy.percentile(arr,p),sep="")
+print(p,"th percentile values of rows are ",numpy.percentile(arr,p,axis=0),sep="")
```
Comment (uttammittal02) [inline ml/Find_Percentile.py]: Here, it's returning the answer for each column and not for rows...kindly check

**Review by uttammittal02 — CHANGES_REQUESTED** [review]

**Comment by himanshusab12** [comment]
I have updated the code please check

**Review by uttammittal02 — APPROVED** [review]
Lgtm

**Pull request merged**